# Request
string scenario_name
---
# Response
bool success
string message